install.sh

```diff
@@ -95,31 +99,34 @@ fi
 
 ask_permission_to_use_homebrew()
 {
-  echo ""
-  echo "Either one or more of the environment variables FC, CC, and CXX are unset or"
-  echo "one or more of the following packages are not in the PATH: pkg-config, realpath, make, fpm."
-  echo "If you grant permission to install prerequisites, you will be prompted before each installation." 
-  echo ""
-  echo "Press 'Enter' to choose the square-bracketed default answer:"
-  printf "Is it ok to use Homebrew to install prerequisite packages? [yes] "
+  cat << EOF | head -c -1
+
+Either one or more of the environment variables FC, CC, and CXX are unset or
+one or more of the following packages are not in the PATH: pkg-config, realpath, make, fpm.
+If you grant permission to install prerequisites, you will be prompted before each installation.
+
+Press 'Enter' to choose the square-bracketed default answer:
+Is it ok to use Homebrew to install prerequisite packages? [yes] 
+EOF
 }
 
 ask_permission_to_install_homebrew()
 {
-  echo ""
-  echo "Homebrew not found. Installing Homebrew requires sudo privileges."
-  echo "If you grant permission to install Homebrew, you may be prompted to enter your password." 
-  echo ""
-  echo "Press 'Enter' to choose the square-bracketed default answer:"
-  printf "Is it ok to download and install Homebrew? [yes] "
+  cat << EOF | head -c -1
+
+Homebrew not found. Installing Homebrew requires sudo privileges.
+If you grant permission to install Homebrew, you may be prompted to enter your password.
+
+Press 'Enter' to choose the square-bracketed default answer:
+Is it ok to download and install Homebrew? [yes] 
+EOF
 }
 
 ask_permission_to_install_homebrew_package()
 {
   echo ""
   if [ ! -z ${2+x} ]; then
-    echo "Homebrew installs $1 collectively in one package named '$2'."
-    echo ""
+    echo -e "Homebrew installs $1 collectively in one package named '$2'.\n"
```

Comment:
`echo -e` is non-POSIX and not available when using `/bin/sh` on many versions of macos, which currently drives this script. 

This patch hunk needs to be removed.

Reply:
Okay I see, I'll make the change.